### ISSUE TITLE
[HL1S] [Steampipe] Map c1a0a Bug 31

### ISSUE THREAD

**Issue by Pinhead-alejandro** [Theuaredead/Half-Life-Source-Update#32]
This is not a bug... but it is incoherent, therefore, you are free to change it or not.

I think that you should repeat this screenshots to show the real views and new source skies:

![2015-07-12_00010](https://cloud.githubusercontent.com/assets/13278870/8638861/3fc24662-28c8-11e5-9954-8373ddc9966b.jpg)

![2015-07-12_00011](https://cloud.githubusercontent.com/assets/13278870/8638864/4f0ad058-28c8-11e5-9abc-07771386abfd.jpg)

![2015-07-12_00012](https://cloud.githubusercontent.com/assets/13278870/8638870/791a5170-28c8-11e5-8641-06c3180923b2.jpg)

![2015-07-12_00013](https://cloud.githubusercontent.com/assets/13278870/8638873/89a49c3a-28c8-11e5-8a16-bca676ae06e1.jpg)

**Comment by Theuaredead** [comment]
Well, I think we'll rather not change these (since like, they were also actually not actually matching what was used in Half-Life 1 (It uses photos from the beta, for the most part)

**Comment by Pinhead-alejandro** [comment]
Mmm I was thinking, and I am confused in any case, with your comment. You said it uses photos from the beta, for the most part. I am sure it is true, but not with the two first pics, since they matches with two places from map c0a0b, the third one match with the old view from map c2a5a and the last view match with the view from map c2a5... I am sure the other pictures like the yellow robot, or that one with the security guard, and the last one with that sentence "working as a team", comes from the beta. Am I wrong? But, did you include the showed pics from here? Because beta or not, they appear in the final version...

**Comment by Theuaredead** [comment]
They come from the beta, all of them do. You can tell by the lighting in the maps, the lighting caused by the skybox becomes more orange than blueish due to the rocks being orange in the beta,
Plus in the apache photo the barney and scientist are missing

**Comment by Theuaredead** [comment]
accidentally reopened due to accidentally sliding my mouse and hitting reopen and comment

**Comment by Pinhead-alejandro** [comment]
Well, yeah I noticed all that since the beginning with the apache, but I imagined that maybe VALVE moved the apache and removed the guys to make that picture like saying this happened some day... but the light thing makes me be agree with you. And no problem with your little accident with the mouse. Maybe I will continue reporting things today, but is hot here. XD But I really want finish this as soon as possible. I have planned finish the game, then report, one issue relationed with the stairs, inform about all the bugs relationed with characters, and finish all with weapons issues, HEV issues, and Gordon Freeman Issues. Then I will repeat all without HD pack, and I will put in the titles something to indicate if I can't find some bug without HD pack, And I will find other things too. After that I will can only test and find more issues if I can... little by little along the game, but you guys will have a, I hope, good list.
